src/utils/address.js

```diff
@@ -1,6 +1,6 @@
 export function shortenAddress (address) {
   const prefix = address.startsWith('0x') ? '0x' : ''
-  return `${prefix}${address.replace('0x', '').substring(0, 4)}...${address.substring(address.length - 4)}`
+  return `${prefix}${address.replace('0x', '').substring(0, prefix ? 6 : 4)}...${address.substring(address.length - 4)}`
```

Comment:
@bradleySuira I did this check here.

Reply:
LGTM

Reply:
Is this used for all addresses? I'm asking because the prefix for other chains could be different. For example in Terra we have is `terra1...`

Reply:
Hi @kraikov, so in the case where there is a prefix, we're leaving it at 4.